src/components/JumbotronVideoComponent.vue

```diff
@@ -9,7 +9,13 @@ const props = defineProps<{
   /**
    * If set, the main element will have a background video.
    */
-  video?: string;
+  videos?: string[];
+  /**
+   * Set the orientation of the spliting.
+   * For the split module
+   * @defaultValue `'vertical'`
+   */
+  splitOrientation?: 'horizontal' | 'vertical';
```

Comment:
Since there are only two possible values, and there's a default value, I propose transforming the property into a boolean. For example, `isHorizontal`. If the prop is `true`, the layout should be displayed horizontally; otherwise, it's vertical. This should simplify the code and improve readability.

*It's a proposal, the current solution also works well and can easily be modified to add other modes if needed :P